### PR TITLE
docs(oas3): indicate updated server support

### PR DESCRIPTION
**Pull request by marcofriso**

### CODE REVIEW

**Comment by kylef** [comment]
@marcofriso is this PR correct? It looks like the same as https://github.com/apiaryio/api-elements.js/pull/444 which adds servers to operation object.

**Comment by marcofriso** [comment]
> @marcofriso is this PR correct? It looks like the same as #444 which adds servers to operation object.

@kylef no, something went wrong, and also my backup disk doesn't work, but it was nothing too complicate, I am going to fix it or write it again

File: packages/fury-adapter-oas3-parser/STATUS.md [hunk 1]
```diff
@@ -24,6 +24,38 @@ Key:
...
+| license | ✓ |
+| version | ✓ |
+
+## Contact Object
+
+| name | ✓ |
+| url | ✓ |
+| email | ✓ |
+
+## Licence Object
```
Comment (kylef) [inline packages/fury-adapter-oas3-parser/STATUS.md:42]:
```suggestion
## License Object
```

License is spent differently in the spec: https://spec.openapis.org/oas/v3.0.2#license-object

File: packages/fury-adapter-oas3-parser/STATUS.md [hunk 2]
```diff
@@ -24,6 +24,38 @@ Key:
...
+
+## Server Object
+
+| url | ✓ |
+| description | ✓ |
+| variables | ✓ |
+
+## Server Variable Object
+
+| enum | ✓ |
```
Comment (kylef) [inline packages/fury-adapter-oas3-parser/STATUS.md:55]:
```suggestion
| Field Name | Support |
|:--|:--|
| enum | ✓ |
```

We're missing the table header rows on all the introduced objects here, could you update them to be consistent with the rest.

File: packages/fury-adapter-oas3-parser/STATUS.md (hunk 1 above)
Comment (marcofriso) [reply packages/fury-adapter-oas3-parser/STATUS.md:42]: sure, changed

File: packages/fury-adapter-oas3-parser/STATUS.md (hunk 2 above)
Comment (marcofriso) [reply packages/fury-adapter-oas3-parser/STATUS.md:55]: changed

**Pull request merged**